.github/actions/isPullRequestMergeable/index.js

```diff
@@ -370,6 +370,22 @@ class GithubUtils {
         });
     }
 
+    /**
+     * Get the most recent workflow run for the given Expensify.cash workflow.
+     *
+     * @param {String} workflow
+     * @returns {Promise}
+     */
+    getLatestWorkflowRunID(workflow) {
```

Comment:
Is there no way to DRY this so its only defined in one place?

Reply:
All these `actions/*/index.js` files are compiled files, so they're bundled with their dependencies. This function is only declared once, in `.github/libs/GithubUtils.js`, but any action that `require`s that dependency will have this code in the compiled action.

Reply:
@roryabraham said these are all compiled files https://github.com/Expensify/Expensify.cash/pull/2495#issuecomment-834751655

Reply:
The build script is [here](https://github.com/Expensify/Expensify.cash/blob/03974374f9e140761147644b8675cab8cfa75122/.github/scripts/buildActions.sh)